apps/emqx/include/emqx_trace.hrl

```diff
@@ -38,4 +38,10 @@
 -define(SHARD, ?COMMON_SHARD).
 -define(MAX_SIZE, 30).
 
+-define(EMQX_TRACE_STOP_ACTION(REASON),
+    {unrecoverable_error, {action_stopped_after_template_rendering, REASON}}
+).
+
+-define(EMQX_TRACE_STOP_ACTION_MATCH, EMQX_TRACE_STOP_ACTION(_)).
```

Comment:
nit: There is no need for this one ?
Just use `EMQX_TRACE_STOP_ACTION(_)` in place.

e.g.
https://github.com/emqx/emqx/pull/12916/files#diff-f56667752bc4971db70fefba01527b860db540ece4edff3a94f77180c9dd6a99R267

Reply:
I tried this first but Elvis complained that I could not use function styled macros in catch expressions.